charlatan/fixture.py

```diff
@@ -65,8 +66,12 @@ def get_parent_values(self):
             elif hasattr(children_value, "update"):
                 # If it's a dict, then we try inheriting from the
                 # parent.
-                new_value = copy.deepcopy(parent_value)
-                new_value.update(children_value)
+                if not self.deep_inherit:
```

Comment:
Why the negative first? If you're going to have logic in both blocks why not make the first one `if self.deep_inherit:`?